### DISCUSSION TITLE
Is the spam/junk folder monitored for training?

### DISCUSSION THREAD

**Discussion by binarydad** [modoboa/modoboa#2445] [General]
I do not know the underlying spam utility, but was wondering if the default junk/mail folder is checked every so often to adjust rules for the spam filter. I know MiaB did this and wondering if modoboa does as well. Thanks.

**Comment by tonioo** [comment]
@binarydad The default setup uses amavis/spamassassin combination so it is possible to activate such a feature but it is not enabled by default since every spam message is put into quarantine.

**Comment by binarydad** [comment]
Got it. Does it learn when messages are flagged as spam or ham from within the quarantine page?

**Comment by binarydad** [comment]
Follow up question, the Quarantine section in the web UI... I can mark messages as spam. Does THAT train the filter? If not, what exactly is "mark as spam" doing besides moving to the Spam/Junk folder? Thanks.
